tests/vmcheck/test-misc-2.sh

```diff
@@ -291,9 +291,17 @@ echo "ok previous staged failure in status"
 
 # Test `deploy --register-driver` option
 # Create and start a transient test-driver.service unit to register our fake driver
-vm_cmd systemd-run --unit=test-driver.service --wait -q \
+vm_cmd systemd-run --unit=test-driver.service -q -r \
        rpm-ostree deploy \'\' \
        --register-driver=TestDriver
+# wait some time for driver to register
+for i in {1..30}
```

Comment:
Not sure if this is good style.
Previously we used the `--wait` option in `systemd-run` so we didn't need this.
However, I've replaced the `--wait` option with `--remain-after-exit` (`-r`) so the fake driver systemd unit is still `active`.

Reply:
We can wait for the daemon to note that it's completed the dummy `Deploy` transaction via the journal.  See callers of `vm_wait_content_after_cursor` - we can look for `Txn Deploy.*successful`.

There's also tooling to use kernel inotify to wait for a path to exist instead of polling.  (This is what systemd `.path` units do).  

But waiting until we see the journal message is polling and race free.

Reply:
Ah thanks! Didn't know there was already a utility function for that.